clean_helpers/map_arabic.py

```diff
@@ -2,4 +2,15 @@ def replace_newline_with_space(batch):
     return {
         **batch,
         "text": [text.replace("\n", " ") for text in batch["text"]]
-    }
\ No newline at end of file
+    }
+
+def remove_html_spans(batch):
+    """Removes lines containing a '<' or '>' from the texts."""
+    bad_strings = ["<", ">"]
+    return {
+        **batch,
+        "text": [
+            "\n".join([line for line in text.split("\n") if not any([bs in line for bs in bad_strings])])
+            for text in batch["text"]
+        ]
+    }
```

Comment:
Hum that's just code duplication with the code one ^^. Ideally it'd be great to have a generic one. Though given the time constraint, maybe it's just better to duplicate code.

Reply:
Yeah, I was a bit reluctant to add modify the existing one without testing it properly. I agree given more time we should just write a "remove html parts" function.

Reply:
No I mean
```
def buid_bad_substring_remover(bs: List[str]):
   def remove_bad_substring(batch):
        return {
            **batch,
            "text": [
                "\n".join([line for line in text.split("\n") if not any([bs in line for bs in bad_strings])])
                for text in batch["text"]
            ]
        }
    return remove_bad_substring
```

Reply:
oh yeah, that's a good idea, let me do that.